src/logion/controllers/locrequest.controller.ts

```diff
@@ -418,7 +418,12 @@ export class LocRequestController extends ApiController {
         const authenticatedUser = await this.authenticationService.authenticatedUserIsLegalOfficerOnNode(this.request);
         const request = await this.locRequestService.update(requestId, async request => {
             authenticatedUser.require(user => user.is(request.ownerAddress));
-            request.accept(moment());
+            if (request.canOpen(request.getRequester())) {
+                request.accept(moment());
```

Comment:
`request.accept(...)` can stay before the condition, only `request.open()` is conditioned.